stl/inc/yvals_core.h

```diff
@@ -1207,7 +1207,9 @@
 #define __cpp_lib_integer_comparison_functions 202002L
 #define __cpp_lib_interpolate                  201902L
 #define __cpp_lib_is_constant_evaluated        201811L
+#define __cpp_lib_is_layout_compatible         202101L

```

Comment:
These values are incorrect. The correct values are noted in #18 (typically they correspond to the month that a feature was voted into the Working Paper); the authoritative source is WG21-N4878 17.3.2 [version.syn], and [SD-FeatureTest](https://isocpp.org/std/standing-documents/sd-6-sg10-feature-test-recommendations) should provide the same info with more context.

Reply:
Resolved